lib/bump/bump.ts

```diff
@@ -71,7 +71,7 @@ export interface AutoBumpOptions {
    * The name of the branch to push the bump commit (e.g. "master")
    * This branch has to exist.
    *
-   * @default - the commit will be pushed to the branch `bump/$VERSION`
+   * @default - the commit will be pushed to the branch `{branchPrefix}/$VERSION`
    */
   branch?: string;
```

Comment:
Maybe rename to `targetBranch`?

Reply:
Maybe, but not worth the breaking change for now. Its going to look different once I finish the refactoring anyway, so lets leave it for now.